libs/sdk-backend-tiger/src/backend/organization/settings.ts

```diff
@@ -18,6 +18,14 @@ export class OrganizationSettingsService
         return this.setSetting("WHITE_LABELING", whiteLabeling);
     }
 
+    public async setTimezone(timezone: string): Promise<void> {
```

Comment:
It seems that we are missing an implementation of `setTimezone` and `setDateFormat` in [sdk-backend-bear](https://github.com/gooddata/gooddata-ui-sdk/tree/master/libs/sdk-backend-bear), aren't we?

→ Please add such implementation to `sdk-backend-bear`.

Reply:
We have already added these functions (setTimezone, setDateFormat) to `sdk-backend-base `, so we don't need to do it in Bear.